api/query_builder.go

```diff
@@ -487,6 +488,8 @@ func RepositoryGraphQL(fields []string) string {
 			q = append(q, "mentionableUsers(first:100){nodes{id,login,name}}")
 		case "projects":
 			q = append(q, "projects(first:100,states:OPEN){nodes{id,name,number,body,resourcePath}}")
+		case "projectsV2":
+			q = append(q, "projectsV2(first:100,query:\"is:open\"){nodes{id,number,title,readme,url}}")
```

Comment:
@babakks : Should the `nodes` fields match the `ProjectV2` struct tags? 🤔 

https://github.com/cli/cli/blob/fc2aec380dd94c6d4eba57aaf075830ccb311f30/api/queries_projects_v2.go#L20-L26

```suggestion
			q = append(q, "projectsV2(first:100,query:\"is:open\"){nodes{id,number,resourcePath,closed,url}}")
```

Reply:
Yeah. Just applied your suggestion.